types/react-sortable-tree/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "private": true,
   "dependencies": {
-    "react-dnd": "^9.3.1"
+    "react-dnd": "^11.1.3"
```

Comment:
Match the `react-dnd` version to `^11.1.3` that `react-sortable-tree` uses. https://github.com/frontend-collective/react-sortable-tree/blob/3c29cbff3edab714d62de577eb1b51f642bc94ec/package.json#L71